packages/core-js/modules/es.promise.js

```diff
@@ -100,9 +100,11 @@ var notify = function (promise, state, isReject) {
               exited = true;
             }
           }
+          var isResultThenable = isThenable(result);
```

Comment:
It's an observable change, so it should be done after the first `if`.

Reply:
Fixed